metadata-io/src/main/java/com/linkedin/metadata/entity/EntityUtils.java

```diff
@@ -90,4 +94,34 @@ public static boolean checkIfRemoved(EntityService entityService, Urn entityUrn)
       return false;
     }
   }
+
+  public static Map<String, Long> convertVersionStamp(String versionStamp) {
```

Comment:
Is there a reason for using string instead of a map directly?

Reply:
This gets sent across the wire and is used directly as an argument in the GraphQL endpoint so this conversion has to happen somewhere. It's easier to pass it through abstraction layers as a String until it gets used.

Reply:
Hmn it might be easier, but is there a chance this will bite us in the future? Having a list of objects with {aspect: string!, version: int!} is more clear than a free form string

Reply:
VersionStamp is a formatted String that looks like: `aspectName:0;aspectName2:1;apectName3:8` but yeah, I get what you're saying. Have updated it, not sure why checkstyle is still failing, the unused imports have been removed :(